src/app/space/settings/areas/areas.component.html

```diff
@@ -1,47 +1,39 @@
+<areas-toolbar
+    (onAddArea)="addChildArea()"
+    (onFilterChange)="filterChange($event)"
+    (onSortChange)="sortChange($event)"
+    [resultsCount]="resultsCount">
```

Comment:
Angular emits warnings like `[Angular] Identifier 'resultsCount' refers to a private member of the component`. This notification exists because AoT will not work if there are any occurrences of html identifiers referring to a private member of the component [1][2]. I think there has been some discussion on seeing if AoT could improve the web application performance [3]. Could you fix these to be public now while you're touching this file?

[1]
https://stackoverflow.com/questions/45520838/identifier-refers-to-a-private-member-of-the-component
[2]
https://github.com/angular/angular/issues/11978
[3] 
https://github.com/openshiftio/openshift.io/issues/2498
https://github.com/openshiftio/openshift.io/issues/938

Reply:
Sure, I'll take a stab at it

Reply:
My IDE currently doesn't show AOT warnings, but I've changed the variables to public.

Reply:
Ah, my bad. s|Angular|Angular Language Service in vscode|

I'm not sure if any other IDEs have similar support.

Reply:
I couldn't get Webstorm to show AOT warnings; although, Joshua can somehow? I read that in Angular 5, "ng test" and "ng build -prod" will report errors when an html template accesses private members.

Other than making private members public, I wasn't able to make any further changes here. Running the build:prod-aot and build:aot targets didn't produce warnings for me.